resources/LetsEncrypt.js

```diff
@@ -0,0 +1,38 @@
+import acme from "acme-client";
```

Comment:
Let's write this in TypeScript, and move it to `app/lib/lets-encrypt.server.ts`, similar to what @Genne23v is doing in #27.

Reply:
I just learned that we should name the file in a certain way in Remix. In case, you don't find this link in my long threaded code review. [Remix docs on server code and bundles](https://remix.run/docs/en/v1/pages/gotchas#server-code-in-client-bundles)